screens/SupportMessageScreen/index.js

```diff
@@ -31,28 +34,22 @@ const SupportMessageScreen = ({navigation}) => {
     }
   };
 
+  const handleChangeText = useCallback((textInput) => {
+    setText(textInput);
+  }, []);
+
   return (
     <View style={styles.supportScreenContainer}>
       <View style={styles.supportScreen}>
         <Header
-          // TODO: This part is will increase later.
           screen="Compose"
           navigation={navigation}
           back="Market"
-          continueTo="Market"
+          continueTo="Profile"
           left="Back"
           right="New"
         />
-
-        <View style={styles.inputContainer}>
-          <TextInput
-            multiline
-            style={styles.input}
-            onChangeText={setText}
-            value={text}
-            placeholder="Compose your message here..."
-          />
-        </View>
+        <SupportMessage handleChangeText={handleChangeText}/>
```

Comment:
you can also send setText instead of handleChangeText

Reply:
ok, i will change it.